### DISCUSSION TITLE
For high Resolution ToBitmap Gives Error of Parameter invalid & slow for Low resolution

### DISCUSSION THREAD

**Discussion by jaydeep6720** [dlemstra/Magick.NET#966] [Help]
```
int newHeight = ImageHeightByRepeatSize(720, ltbRepeat, OpenedFile.Height);
MagickGeometry Newsize = new MagickGeometry(OpenedFile.Width, newHeight);
Newsize.IgnoreAspectRatio = true;
label1.Text = Convert.ToString(OpenedFile.Density);
OpenedFile.Sample(Newsize);
label2.Text = Convert.ToString(OpenedFile.Height);
label3.Text = Convert.ToString(OpenedFile.Width);
pictureBox1.Image = OpenedFile.ToBitmap();
pictureBox1.SizeMode = PictureBoxSizeMode.Zoom; 
```

> This is Simple code which resize image in Height only : width dont needed  to be changed
File details :
File -1 : Tiff file ( Width 23400 pxl Height 9072 pxl Density 360x360 .... 
                         Process time with ToBitmap : 13 sec, without ToBitmap : 3.5 Sec
File-2 :  Tiff file (width  46800 pxl Height 18144 pxl Density 720x720....
                         With ToBitmap : Not working , Without ToBitmap : 4.2 sec
File Download link is valid only for 7 days : 
Link : [https://we.tl/t-GTtDkIEubm]
tried so many things to low down time duration & physicality but not succeeded. 

Update ;
Target Pixel Height is 12954 for 360 Density & 25908 for 720 Density

**Comment by dlemstra** [comment]
Which version of Magick.NET are you using? And please attach the file to this issues.

**Reply by jaydeep6720** [reply]
Magick.Net.Core 7.0.1
Magick.NET.SystemDrawing 2.0.17
Magick.NET-Q16-x64 7.24.1

[https://we.tl/t-GTtDkIEubm](url)

File is too big to put it & format is tiff which is not supported here.

**Comment by dlemstra** [comment]
You can zip the individual files:

[720 Density.zip](https://github.com/dlemstra/Magick.NET/files/6671947/720.Density.zip)
[360Density.zip](https://github.com/dlemstra/Magick.NET/files/6671948/360Density.zip)

**Reply by jaydeep6720** [reply]
@dlemstra Hello Sir
Please check this issue when you get free time...!!

**Reply by dlemstra** [reply]
It looks like you are running into the limits of the `Bitmap` class. Those dimensions are probably not possible with that class.
